docs/assets/img/nextjs-sitecore-analytics.svg

```diff
@@ -1,3 +1,3 @@
atures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 153px; height: 1px; padding-top: 765px; margin-left: 287px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; max-height: 96px; overflow: hidden; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><ul><li>cookie</li><li>user-agent</li><li>referer</li><li>X-Forwarded-For</li></ul></div></div></div></foreignObject><text x="287" y="769" fill="#000000" font-family="Helvetica" font-size="14px">cookieuser-agentrefere...</text></switch></g><rect x="285.46" y="815" width="155" height="60" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 153px; height: 1px; padding-top: 845px; margin-left: 287px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; max-height: 56px; overflow: hidden; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><ul><li>set-cookie</li></ul></div></div></div></foreignObject><text x="287" y="849" fill="#000000" font-family="Helvetica" font-size="14px">set-cookie</text></switch></g><path d="M 316.37 565 L 398.63 565" fill="none" stroke="#999999" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 311.12 565 L 318.12 561.5 L 316.37 565 L 318.12 568.5 Z" fill="#999999" stroke="#999999" stroke-miterlimit="10" pointer-events="all"/><path d="M 403.88 565 L 396.88 568.5 L 398.63 565 L 396.88 561.5 Z" fill="#999999" stroke="#999999" stroke-miterlimit="10" pointer-events="all"/><path d="M 362.96 657 L 550 590" fill="none" stroke="#1ca6a3" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 165 183.63 L 165 156.37" fill="none" stroke="#999999" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 165 188.88 L 161.5 181.88 L 165 183.63 L 168.5 181.88 Z" fill="#999999" stroke="#999999" stroke-miterlimit="10" pointer-events="all"/><path d="M 165 151.12 L 168.5 158.12 L 165 156.37 L 161.5 158.12 Z" fill="#999999" stroke="#999999" stroke-miterlimit="10" pointer-events="all"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
The diagram shows `RestLayoutService`. We can't use GraphQL for this?

Reply:
@AncaIO That's correct. Currently this tracking and analytics architecture is only compatible with the REST-based Layout Service. I've got some bits about that on the analytics docs (\docs\nextjs\tracking-and-analytics\configuration and \docs\nextjs\tracking-and-analytics\overview) and just pushed an update for the "How is Next.js Different" doc (\docs\nextjs\getting-started-nextjs\how-is-nextjs-different). Maybe this needs to be more pronounced though?